### DISCUSSION TITLE
About DogeCoin NFT

### DISCUSSION THREAD

**Discussion by Jassi10000-zz** [dogecoin/dogecoin#2763] [General]
If I make a DogeCoint NFT , would it be sold in nice amt. of ether or its overrated ?

**Comment by rnicoll** [comment]
The technology for how NFTs are minted and tracked on Dogecoin is being developed by an external developer, and beyond that I don't think anyone's actually worked out the market/trading model for them yet, no.

**Comment by Jassi10000-zz** [comment]
Since I can see , there's a lot of problem , how the hell people are then so cool bout this ?
